utils/currency.ts

```diff
@@ -3,7 +3,8 @@ export type Currency = {
   name: string
   symbol: string
   decimals: number
-  logo: string
+  logo?: string
```

Comment:
Let's create a util function that takes Currency as an argument and return the logo url.